csds-client/client/client.go

```diff
@@ -232,7 +235,7 @@ func (c *Client) doRequest(streamClientStatus csdspb.ClientStatusDiscoveryServic
 	}
 
 	// post process response
-	if err := printOutResponse(resp, c.info.configFile); err != nil {
+	if err := printOutResponse(resp, c.info.configFile, c.info.visualization, c.info.monitorInterval != 0); err != nil {
```

Comment:
OK, so can we pass `c.info` to `printOutResponse`?

Reply:
Modified these to `c.info`